packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -231,11 +231,11 @@ Will double processing times. Use only for debugging purposes.",
     group: "chain",
   },
 
-  "chain.stateArchiveMode": {
+  "chain.archiveMode": {
```

Comment:
curious about the rationale for renaming, so we are combining block and state archive now?

Reply:
We want `ArchiveStore` to have only one `mode` and handles any types of archives based on that mode. So why stripped the `state` from there.